skimage/feature/peak.py

```diff
@@ -159,3 +159,73 @@ def peak_local_max(image, min_distance=10, threshold_abs=0, threshold_rel=0.1,
         nd_indices = tuple(coordinates.T)
         out[nd_indices] = True
         return out
+
+
+def get_scale_local_maximas(cube_coordinates, laplacian_cube):
```

Comment:
https://gist.github.com/warmspringwinds/1aa093ddc6f5b5ed48a6

You can see the benchmarks there.
I will be really glad if you help me.

But I think that the function is called only once in my method and won't speed up the whole script really much if we improve it further.